### DISCUSSION TITLE
👋 Welcome to Geist Discussions!

### DISCUSSION THREAD

**Discussion by unix** [zeit-ui/react#431] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by moatorres** [comment]
Hey @unix, just passing by to let you know that I think Geist UI is awesome. Glad to collaborate in any meaningful way, just LMK and keep up with the great work.

**Reply by moatorres** [reply]
By the way, I may have found a typo on [here](https://github.com/geist-org/react/blob/6da0509316d294ec478075bb9d187ccfd230582c/components/utils/prop-types.ts) on line 44:

```js
const copyTypes = tuple('default', 'slient', 'prevent') // --> 'slient'
```

Wondering if it should be `silent` instead of `slient`?

**Reply by unix** [reply]
😿  Yes, `silent` should be used here, a subsequent update will fix this typo.
